### DISCUSSION TITLE
Future of OAuth / OpenID Connect support in Open Source / OWASP DefectDojo

### DISCUSSION THREAD

**Discussion by J12934** [DefectDojo/django-DefectDojo#9628] [Q&A] [answered]
Hi 👋

I saw on the DefectDojo pricing site (https://www.defectdojo.com/pricing) that OAuth and SAML support is considered a "Dojo Pro" feature and not part of the "OWASP" plan? 

Does this mean the existing OAuth / SAML support will be removed from the OpenSource project?
https://documentation.defectdojo.com/integrations/social-authentication/

**Comment by manuel-sommer** [comment, marked as answer]
Hi @J12934, 
No, rather that the maintainers help you with OAuth / SAML directly in "Dojo Pro", basically configuring it for you. Correct me, if I am wrong @mtesauro .

**Reply by mtesauro** [reply]
@J12934 manual-sommer is correct - there's many nuances and configuration options when setting up SSO to any application.

If you're willing to handle those issues yourself, there's nothing stopping you from setting up SSO with DefectDojo.

There's several things in DefectDojo Pro that reduce or eliminate that work in DefectDojo Pro.
